app/components/UI/NetworkVerificationInfo/NetworkVerificationInfo.tsx

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import { Linking } from 'react-native';
+import { strings } from '../../../../locales/i18n';
+import { CommonSelectorsIDs } from '../../../../e2e/selectors/Common.selectors';
+import Text, {
+  TextVariant,
+} from '../../../component-library/components/Texts/Text';
+
+import PickerNetwork from '../../../component-library/components/Pickers/PickerNetwork';
+import Accordion from '../../../component-library/components/Accordions/Accordion';
+import Banner, {
+  BannerAlertSeverity,
+  BannerVariant,
+} from '../../../component-library/components/Banners/Banner';
+import { DEFAULT_BUTTONLINK_LABEL_COLOR } from '../../../component-library/components/Buttons/Button/variants/ButtonLink/ButtonLink.constants';
+import { useStyles } from '../../../component-library/hooks';
+import styleSheet from './NetworkVerificationInfo.styles';
+import { CustomNetworkInformation } from './NetworkVerificationInfo.types';
+import { ScrollView } from 'react-native-gesture-handler';
+
+/**
+ * NetworkVerificationInfo component
+ */
+const NetworkVerificationInfo = ({
+  customNetworkInformation,
+}: {
+  customNetworkInformation: CustomNetworkInformation;
+}) => {
+  const [networfInfoMaxHeight, setNetworkInfoMaxHeight] = useState<
+    number | null
+  >(null);
+  const [networkDetailsExpanded, setNetworkDetailsExpanded] = useState(false);
+  const { styles } = useStyles(styleSheet, {});
+
+  const renderNetworkInfo = () => (
+    <ScrollView
+      nestedScrollEnabled
+      style={[
+        styles.accountCardWrapper,
+        networfInfoMaxHeight ? { maxHeight: networfInfoMaxHeight } : undefined,
```

Comment:
Typo and you can use a spread here:
```suggestion
        ...(networkInfoMaxHeight && { maxHeight: networkInfoMaxHeight }),
```

Reply:
typo fixed! I don't think we can spread it since it's an array 

```
   style={[
        styles.accountCardWrapper,
        networkInfoMaxHeight ? { maxHeight: networkInfoMaxHeight } : undefined,
      ]}
```